pkg/orchestrator/etcd_worker.go

```diff
@@ -80,12 +84,22 @@ func (worker *EtcdWorker) Run(ctx context.Context, timerInterval time.Duration)
 
 	watchCh := worker.client.Watch(ctx1, worker.prefix.String(), clientv3.WithPrefix())
 	var pendingPatches []*DataPatch
+	var exiting bool
+	var sessionDone <-chan struct{}
```

Comment:
Maybe put these three declarations in a pair of parentheses following `var`? Rest LGTM